projects/components/src/radio/radio-group.component.ts

```diff
@@ -39,6 +40,9 @@ export class RadioGroupComponent implements OnInit {
   @Input()
   public disabled: boolean | undefined;
 
+  @Input()
+  public optionsDirection: OptionsDirection = OptionsDirection.Column;
```

Comment:
this is an input within the radiogroup component. so it can use a simpler name. perhaps just `orientation` or `direction`?

Reply:
not a bad update to make (I'd opt for orientation, and update the enum too to something like RadioGroupOrientation), worth doing before this gets consumed anywhere.